group_vars/development

```diff
@@ -3,10 +3,10 @@ mysql_root_password: devpw
 web_user: vagrant
 
 wordpress_sites:
-  example.dev:
+  example.com:
```

Comment:
Ok, yeah, this discussion gives me clarity on the distinction between site name and host names in group_vars. I had them all in one hazy mental mess.